src/app/examples/modal-example/modal-example.component.tns.html

```diff
@@ -0,0 +1,3 @@
+<StackLayout>
+    <Button text="open modal" (tap)="openModal()"></Button>
```

Comment:
Not of any importance what so ever... but (and this may be me suffering infinitesimally from O.C.D) can we upper case the 'o' in "open modal"?